docs/provision_azure_iothub.md

```diff
@@ -0,0 +1,44 @@
+# Provision Azure IoT Hub #
+
+## Overview ##
+
+When provisioning Azure Services in **Azure IoT Device Workbench**, the common provision process includes the provision of **Azure IoT Hub**. 
+
+  ![provision process](pic/provision_process.png)
+
+This document contains the detailed steps on how to provision Azure IoT Hub.
+
+## Steps for Provision Azure IoT Hub ##
+
+1. In the solution window, open the command palette and select **Azure IoT Device Workbench: Provision Azure Services...**.
+
+	![IoT Device Workbench: Cloud -> Provision](pic/iot-workbench-cloud-provision.png)
+
+1. If you haven't login to Azure in Visual Studio Code before, the following will be pop-up to guide you for Azure sign-in. 
```

Comment:
If already login then jump to section 5?

Reply:
hmm... can combine step 2 & 3 in one step, mention skip this step if already login